<ui version="4.0" >
 <class>RSS</class>
 <widget class="QWidget" name="RSS" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QSplitter" name="splitter_h" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="listStreams" >
         <property name="contextMenuPolicy" >
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RSS streams:</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>2</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="delStream_button" >
           <property name="minimumSize" >
            <size>
             <width>55</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Delete selected streams</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshAll_button" >
           <property name="minimumSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Refresh RSS streams</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addStream_button" >
           <property name="minimumSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Add a new RSS stream</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="iconSize" >
            <size>
             <width>55</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>listStreams</zorder>
      <zorder></zorder>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="news_lbl" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>&lt;b>News:&lt;/b> &lt;i>(double-click to open the link in your web browser)&lt;/i></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_v" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QListWidget" name="listNews" />
         <widget class="QTextBrowser" name="textBrowser" />
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionDelete" >
   <property name="text" >
    <string>Delete</string>
   </property>
  </action>
  <action name="actionRename" >
   <property name="text" >
    <string>Rename</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="text" >
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionCreate" >
   <property name="text" >
    <string>Add RSS stream</string>
   </property>
  </action>
  <action name="actionRefreshAll" >
   <property name="text" >
    <string>Refresh all streams</string>
   </property>
  </action>
  <action name="actionMark_all_as_read" >
   <property name="text" >
    <string>Mark all as read</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
